tests/clean-snapshots.R

```diff
@@ -28,6 +28,25 @@ edit_markdown <- function (md) {
     i <- grep ("An interactive visualisation", md)
     md [i] <- gsub ("\\]\\(.*\\)", "](network.html)", md [i])
 
+    # remove <details> sections of function usage because numbers of function
+    # calls to dependency packages can be artibrarily ordered when numbers of
+    # calls are equal.
+    i <- grep ("^<details>", md)
+    j <- grep ("</details>", md)
+    # These are the sub-sections:
+    index <- which (j [-length (i)] > i [-1])
+    if (length (index) > 0L) {
+        index <- seq (min (i [index + 1]), max (j [index]))
+        md <- md [-index]
+    }
+
+    # and for some reason, the covr environment pulls in an extra external
```

Comment:
yeah, `covr` is a nightmare for software which intro-spects packages, because it changes the underlying source code. And does so in a cloned environment of the whole package, so lots of package-environment expectations also break.